### PR TITLE
gnunet 0.19.2

### PR DESCRIPTION
**Pull request by p-linnane**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by p-linnane** [comment]
Same build failure as previous attempts:

```
  gnunet-rest-server.c:135:12: error: use of undeclared identifier 'LOGIN_NAME_MAX'
  char cuser[LOGIN_NAME_MAX];
             ^
  gnunet-rest-server.c:1210:35: error: use of undeclared identifier 'LOGIN_NAME_MAX'
        if (0 != getlogin_r (cuser, LOGIN_NAME_MAX))
                                    ^
  2 errors generated.
```

**Comment by p-linnane** [comment]
Opened an issue upstream: https://bugs.gnunet.org/view.php?id=7557

File: Formula/gnunet.rb [hunk 1]
```diff
@@ -35,9 +32,9 @@ class Gnunet < Formula
 
   def install
     ENV.deparallelize if OS.linux?
-    # Workaround for Xcode 14 ld until Makefile is fixed
-    # Ref: https://github.com/Homebrew/homebrew-core/pull/113789#issuecomment-1288125182
-    ENV.append_to_cflags "-fuse-ld=lld" if DevelopmentTools.clang_build_version >= 1400
+    # Workaround for LOGIN_NAME_MAX not defined on macOS
+    # Ref: https://bugs.gnunet.org/view.php?id=7557
+    ENV.append_to_cflags "-DLOGIN_NAME_MAX=255" if OS.mac?
```
Comment (cho-m) [inline Formula/gnunet.rb:37]: Probably shouldn't merge this unless can confirm the value across macOS systems. It is the value on my ARM Ventura machine:
```console
❯ getconf LOGIN_NAME_MAX
255
```

Perhaps using `sysconf(_SC_LOGIN_NAME_MAX)` would be a more accurate alternative.

Comment (p-linnane) [reply Formula/gnunet.rb:37]: I can spin VM's for Big Sur and Monterey arm64 to test, but I don't own any Intel machines anymore.

Comment (p-linnane) [reply Formula/gnunet.rb:37]: Monterey arm64 value is 255.

**Review by SMillerDev — DISMISSED** [review]

**Comment by p-linnane** [comment]
Added a patch from upstream that worked for me locally.

**Review by SMillerDev — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**